### PR TITLE
More QoL changes

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request
- Sleepers have more attractive chems as upgrades
- Hyper sleeper has even more amazing selection of chems to justify their rarity
- Tobacco now tastes like tobacco, with a distinction between cigar's fine tobacco and cigarette tobacco
- Makes Peridaxon no longer heal bones too
- Fixes to Ryetalyn so that it continues to proc the mutations cleanup on every life tick
- Fixes Paracetamol taking ages to metabolize
- Adds an overdose effect for Noexcutite
- Changes Peridaxon so that it no longers heals bones, to justify making Ossisine (also Peri shouldn't heal bones!)

## Changelog
:cl:DimmaDunk
add: Added Fine Nicotin for the expensive cigars
add: Added Noexcutite overdose effect
tweak: Peridaxon no longer heals bones, only internal organs
balance: Changed the Sleeper's unlockable chems by tier, as well as the Hyper Sleeper's chemical selection
fix: Ryetalyn should now also work on every life tick instead of having to wait for it to metabolize out, then re-dose for it to affect only when first added to the system
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagents/drugs.dm [hunk 1]
```diff
@@ -176,9 +176,36 @@
...
 	M.add_side_effect("Headache", 11)
 	if(prob(5))
-		M.vomit()
+		M.emote("cough")
+	M.adjustOxyLoss(0.5)
 	M.adjustToxLoss(0.5)
 
+/datum/reagent/drug/nicotineplus
+	name = "Nicotine"
+	id = "nicotine"
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/drugs.dm:185]:
```suggestion
	id = "nicotineplus"
```

File: code/modules/reagents/reagents/drugs.dm [hunk 2]
```diff
@@ -176,9 +176,36 @@
 /datum/reagent/drug/nicotine/overdose(var/mob/living/carbon/M, var/alien)
 	M.add_side_effect("Headache", 11)
 	if(prob(5))
-		M.vomit()
+		M.emote("cough")
+	M.adjustOxyLoss(0.5)
 	M.adjustToxLoss(0.5)
 
+/datum/reagent/drug/nicotineplus
+	name = "Nicotine"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/drugs.dm]:
```suggestion
	name = "Fine Nicotine"
```

**Review by ThePainkiller — COMMENTED** [review]
I swear in my locale it was changed too, odd

File: code/modules/reagents/reagents/medicine.dm [hunk 3]
```diff
@@ -453,22 +453,26 @@
...
+    M.disabilities = 0
+    M.sdisabilities = 0
 
-	// Might need to update appearance for hulk etc.
-	if(needs_update && ishuman(M))
-		var/mob/living/carbon/human/H = M
-		H.update_mutations()
-		H.update_body() //Don't let husks stay wrinkly all the time, we gotta fix them!
+    // Might need to update appearance for hulk etc.
+    if(needs_update && ishuman(M))
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/medicine.dm:472]: I DID NOT EVEN TOUCH THIS WHY IS THIS SPACES INSTEAD OF TABULATIONS, I ONLY ADDED LINE 462

**Pull request merged**